webpack/components/Table/TableWrapper.js

```diff
@@ -190,6 +192,11 @@ const TableWrapper = ({
           <FlexItem style={{ marginLeft: '16px' }}>
             {actionButtons}
           </FlexItem>}
+        {showToggleGroup &&
+          <FlexItem style={{ marginLeft: '16px' }}>
+            {toggleGroup}
+          </FlexItem>}
```

Comment:
I'm wary of adding yet another prop to TableWrapper. What about just rendering the toggle group in `actionButtons`? @sjha4 @Andrewgdewar thoughts?